meta-resin-common/recipes-support/resin-mounts/resin-mounts/etc-machine-id.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Machine-id Mount Point
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=systemd-machine-id-commit.service
+
+[Service]
```

Comment:
Commenting for reference: We discussed and decided to separate the unit into a normal unit that only `mkdir`s and `touch`es the file and a normal `.mount` unit that will depend on the first unit